math/src/staking/math.rs

```diff
@@ -37,9 +37,9 @@ pub fn calculate_slashed_points(
 	stake_weight: u8,
 ) -> Option<Balance> {
 	let stake_weighted = current_stake.checked_mul(stake_weight.into())?;
-	FixedU128::checked_from_rational(stake_increase, stake_weighted)?
-		.min(FixedU128::one())
-		.checked_mul_int(points)
+	let p = stake_increase.checked_mul(points)?;
```

Comment:
The math here is correct and I believe decreases rounding error.

In Omnipool calculations when we do this we convert to U256 so that when we multiply the two U128s we do not get overflow. We do not appear to be doing that here. Did we consider the magnitudes of the quantities involved and decide that since the points quantity is low we do not need to convert to U256? Won't we run into issues as people continue to accrue points over time?